### DISCUSSION TITLE
custom model dosent show up in game

### DISCUSSION THREAD

**Discussion by tiffbu** [TartaricAcid/TouhouLittleMaid#529] [Q&A]
i used the blockbench plugin to made a pack with one maid model in it, and put the resorce pack into game and actvited it but it dosent show up on the change skin page in game. i tried opening the files and check if everything is right and also tried to use conversion in blockbench to make the model "bedrock legacy model". what could be the problem?

**Comment by TartaricAcid** [comment]
I don't know either, sorry
